sphinx/tutorial/rose/suites.rst

```diff
@@ -416,8 +416,9 @@ See the :ref:`Cheat Sheet` for more information.
       Running :ref:`command-rose-suite-run` will cause the suite to be
       installed, validated and run.
```

Comment:
```suggestion
```

Reply:
I hadn't been going to be that brutal, but yes, that works!